src/riak_ql_lexer.xrl

```diff
@@ -188,18 +190,27 @@ clean_up_hex([$0,$x|Hex]) ->
     end.
 
 clean_up_literal(Literal) ->
-    RemovedOutsideQuotes = accurate_strip(Literal, $'),
-    DeDoubledInternalQuotes = re:replace(RemovedOutsideQuotes,
-                                         "''", "'",
-                                         [global, {return, list}]),
-    list_to_binary(DeDoubledInternalQuotes).
-
-strip_quoted(QuotedString) ->
-    % if there are unicode characters in the string, throw an error
-    [error(unicode_in_quotes) || U <- QuotedString, U > 127],
-
-    StrippedOutsideQuotes = accurate_strip(QuotedString, $"),
-    re:replace(StrippedOutsideQuotes, "\"\"", "\"", [global, {return, binary}]).
```

Comment:
Guessing you looked at the `re:replace` performance which is why you wrote your own?